server/routes/api/auth/userAuth.js

```diff
@@ -0,0 +1,21 @@
+import { Router } from 'express';
+import authController from '../../../controllers/auth/userAuth';
+import
+  emailValid from '../../../middlewares/signupValidations/emailValidation';
+import
+  pwdValid from '../../../middlewares/signupValidations/passwordValidation';
```

Comment:
I think all validations should be in one folder. So we can have `validations/emailValidation`. Why? `emailValidation` can also be required in another action that is not a signup.